doc/api/child_process.markdown

```diff
@@ -755,7 +784,7 @@ grep.stdin.end();
 * `message` {Object}
 * `sendHandle` {Handle object}
 * `callback` {Function}
-* Return: Boolean
+* Return: Boolean | Promise
```

Comment:
Very uncomfortable with this kind of returns something or Promise kind of pattern. I would rather have a `sendAsync` kind of variant that always returns a Promise and keep `send` as it is. Overloading return values can get become problematic for developers who aren't careful.